feg/gateway/services/aaa/servicers/accounting.go

```diff
@@ -315,6 +315,21 @@ func createSessionOnSessionManager(mac net.HardwareAddr, subscriberId *lte_proto
 		RatType:         lte_protos.RATType_TGPP_WLAN,
 		HardwareAddr:    mac,
 		RadiusSessionId: aaaCtx.GetSessionId(),
+		CommonContext: &lte_protos.CommonSessionContext{
```

Comment:
Can you leave a `//TODO:` here or at line 310 with a note reminding that the repeated fields need to be removed once we transitioned?  This way it will be easier to find them. Also it helps other people whp may look. at the code  while we are doing the transition

Reply:
Will do